filehandling.c

```diff
@@ -1,12 +1,27 @@
 #include<stdio.h>
 
 struct employee
-{    char employeeName[50];    int employeeId;
+{ 
+	   char employeeName[50]; 
+	   int employeeId;
 };
 
 int main()
-{    FILE *fp=NULL;    int n;    printf("enter number of employee:");    scanf("%d",&n);    struct employee emp[n];    int i;    for(i=0;i<n;i++)
-{    printf("Enter the employee details\n");    printf("Name:");    scanf("%s",emp[i].employeeName);    printf("Id:");    scanf("%d",&emp[i].employeeId);
+{
+	    FILE *fp=NULL;
```

Comment:
add space before or after each operator
 FILE *fp = NULL;

Reply:
What will happen if employeeName is longer than 50 char?

Reply:
If employeeName is longer than 50 characters, it may be an error message may be displayed.
 String variables are not used to store employee names because strings may contain a variety of characters such as numbers and symbols, which can lead to data entry errors. it store only alphanumeric characters, data type to use for employee names would be a character array